Tests/XcodeGenKitTests/SourceGeneratorTests.swift

```diff
@@ -517,6 +517,23 @@ class SourceGeneratorTests: XCTestCase {
                 try expect(sourcesBuildPhase.files.count) == 1
             }
 
+            $0.it("add only carthage dependencies with same platform", closure: {
```

Comment:
Can this be made a trailing closure

Reply:
Done in ec60901ff749383943ced49e3b77aeb60c50603a.